src/reducer.js

```diff
@@ -565,6 +543,11 @@ export const onTokenFetchHistoryRequested = (state, action) => {
   };
 };
 
+export const onStartWalletNotStarted = (state) => ({
```

Comment:
Why not `setWalletNotStarted`?

Reply:
Actually, we don't need this action and reducer anymore